docs/developer-docs/daos/nns/using-the-nns-dapp/nns-dapp-importing-tokens.mdx

```diff
@@ -0,0 +1,67 @@
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/zcXmarBk2L4?si=OIxq9LwY0DLzHvE1" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+
+## Overview
+
+The NNS dapp focuses on providing a simple user interface where community members can interact with different on-chain governance systems deployed on the Internet Computer.
+
+To extend wallet capabilities, users of the NNS dapp can now import any token that supports ICRC-1 no longer limited to ICP and SNS governance tokens. This includes all ICRC-2 and ICRC-3 tokens.
+
+## How to find canister IDs
+
+There are several ways to find ledger canister IDs of ICRC-1 tokens. The tutorial video above mentions two. This is a more comprehensive list.
+
+- [ICPSwap](https://info.icpswap.com/swap): Curated list of tokens traded on ICPSwap updated by the ICPSwap SNS DAO.
+- [ICP Tokens](https://icptokens.net/): Curated list of tokens traded on ICP-based DEXs updated by ICP Tokens.
+- [ICP Dashboard](https://dashboard.internetcomputer.org/): List of all chain-key tokens (under Chain Fusion menu option) updated automatically.
+- [nftGeek](https://t5t44-naaaa-aaaah-qcutq-cai.raw.icp0.io/tokens): Curated list of tokens traded on ICP-based DEXs updated by nftGeek.
+- [ICPEx](https://icpex.org/dashboard): Curated list of tokens traded on ICPEx updated by ICPEx.
+- [CoinMarketCap](https://coinmarketcap.com/): The ICRC tokens that are listed on CoinMarketCap provide their ledger canister ID under Contract.
+- [CoinGecko](https://coingecko.com/): The ICRC tokens that are listed on CoinGecko provide their ledger canister ID under Contract.
+
+
+## How to import ICRC tokens
+
+First, find the ledger canister ID of the token you want to import to your NNS dapp wallet. The list above covers most tokens deployed on the Internet Computer. Once you have its ledger canister ID, you may optionally look for its index canister to display the transaction history.
+
+:::info
+Importing an index canister is completely optional. Many popular tokens **do not have index canisters**.
+:::
+
+- **Step 1:**  Sign into the NNS dapp, and navigate to **Tokens**. At the bottom of the table, click **Import Token**.
+
+![NNS Import Tokens 1](../../_attachments/nns-import-tokens-1.webp)
+
+:::info
+If you don't want to see tokens you don't hold, you can click the settings button at the top right corner of the table, and enable **Hide zero balances**.
+:::
+
+- **Step 2:** Paste the ledger and index canister IDs to the corresponding input fields. If you don't provide the index canister ID now, you can add it any time in the future.
+
+![NNS Import Tokens 2](../../_attachments/nns-import-tokens-2.webp)
+
+- **Step 3**: Confirm that this is the token you want to import. By clicking the rightmost button, you can see the canister on the ICP dashboard.
```

Comment:
I don't follow what you mean by the second sentence "the rightmost button". Even when looking at the picture